tests/e2e_logging.py

```diff
@@ -1588,6 +1588,61 @@ def test_committed_index(network, args):
     }
 
 
+@reqs.description("Test the commit endpoints view_history feature")
+def test_commit_view_history(network, args):
```

Comment:
It seems that this test would make more sense in `election_test.py`? And possible worth included in the `test_suite.py` too?

Reply:
Moved, seems it can be a bit flaky now, any ideas as to how I should make it more robust to the leader changing in-between requests.

Reply:
Seems it is when the transaction id changes (just seqno not view) so will just ignore txid in these tests.